client/src/translations/index.tsx

```diff
@@ -0,0 +1,682 @@
+        const a = A.differences.total;
+        const b = B.differences.total;
+        return reverse * (b - a);
+      } else if (sort === "mdn_url") {
+        const a = A.mdn_url;
+        const b = B.mdn_url;
+        return reverse * a.localeCompare(b);
+      } else {
+        throw new Error(`Unrecognized sort '${sort}'`);
+      }
+    });
+  const pageCount = Math.ceil(counts.found / pageSize);
+
+  return (
+    <div className="documents">
+      <h3>
+        Documents found ({filteredDocuments.length.toLocaleString()}){" "}
+        {page > 1 && <span className="page">page {page}</span>}{" "}
+        <small>of {counts.total.toLocaleString()} in total</small>
+      </h3>
+
+      <table>
+        <thead>
+          <tr>
+            <TH id="mdn_url" title="Document" />
+            <TH id="popularity" title="Popularity" />
+            <TH id="modified" title="Last modified" />
+            <TH id="differences" title="Differences" />
+          </tr>
+        </thead>
+        <tbody>
+          {filteredDocuments
+            .slice((page - 1) * pageSize, page * pageSize)
+            .map((doc: Document) => {
+              return (
+                <tr key={doc.mdn_url}>
+                  <td>
+                    <Link
+                      to={`${doc.mdn_url}#_flaws`}
+                      title={doc.title}
+                      target="_blank"
+                    >
+                      {showBriefURL(doc.mdn_url)}
+                    </Link>
+                    <br />
+                    <span className="document-title-preview">
+                      {filterTitle
+                        ? getHighlightedText(doc.title, filterTitle)
+                        : doc.title}
+                    </span>
+                  </td>
+                  <td
+                    title={
+                      doc.popularity.ranking
+                        ? `Meaning there are ${
+                            doc.popularity.ranking - 1
+                          } more popular pages than this`
+                        : "Meaning it has no ranking. Most likely a very rare (or new) document"
+                    }
+                  >
+                    {!doc.popularity.ranking
+                      ? "n/a"
+                      : `${getGetOrdinal(doc.popularity.ranking)}`}{" "}
+                    <small title="For the parent document">
+                      (
+                      {!doc.popularity.parentRanking
+                        ? "n/a"
+                        : `${getGetOrdinal(doc.popularity.parentRanking)}`}
+                      )
+                    </small>
+                  </td>
+                  <td>{showLastModified(doc)}</td>
+                  <td>
+                    {doc.differences.count ? (
+                      <>
+                        {doc.differences.count.toLocaleString()}{" "}
+                        <small>
+                          ({doc.differences.total.toLocaleString()} total)
```

Comment:
I haven't looked into the details, but I'm confused why the `doc.differences.count` always seems to be `0` or `1`, even when the document has more than one translation difference. To me, it seems clearer to only show the total number of translation differences for each document, so instead of `1 (4 total)` (see below):

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/3743693/118198962-d9188b80-b406-11eb-8701-c82fc00743de.png">

instead show `4`. Or maybe I'm misunderstanding something?

Reply:
The explanation is that today there's only 1 type of difference; `macro`. 
E.g. 
```js
var differences = {
  'macro': [
    // KS macro difference 1,
    // KS macro difference 2,
  ]
}
```
The idea is that there will be more types. Like `codeblock` or `headings` or `front-matter`. 
So you might have 3 KS macro differences and 2 codeblock differences. Should it say 5 then? 
Yes, I think it should :)